### DISCUSSION TITLE
Chinese simplified doesn't take into account of how Chinese keyboards work

### DISCUSSION THREAD

**Discussion by ThisisShoo** [monkeytypegame/monkeytype#4900] [Ideas]
Chinese sentences rarely contain spaces between words, unlike in other languages. Therefore, it's counterintuitive to place spaces between words. I recommend adding a feature where the space is automatically added/skipped since naively merging the words together will only create a wordsoupthatlookslikethis.

I would also suggest using coherent or incoherent sentences (basically, words separated by punctuations) instead of discrete words, but that seems to edge on defeating the whole test's purpose.

**Comment by Miodec** [comment]
You can use the "nospace" funbox for that.
